### DISCUSSION TITLE
Unpackrr

### DISCUSSION THREAD

**Discussion by kloknibor** [tteck/Proxmox#1059] [Ideas]
Hi guys,

I think the following app might add nicely to our current media portfolio:

https://github.com/Unpackerr/unpackerr

in case an archive is downloaded unpackrr will automatically unpack the download and make it ready for import.

what do you guys think?

**Comment by leonelafonsoe** [comment]
Hey! I've just found a use case where this would be perfect. I could try a pull request with this if its useful. Anybody else?

**Reply by tteck** [reply]
It is as easy as:
```
wget -q https://github.com/Unpackerr/unpackerr/releases/download/v0.12.0/unpackerr_0.12.0-547_amd64.deb
dpkg -i unpackerr_0.12.0-547_amd64.deb
```
To modify the config file:
```
nano /etc/unpackerr/unpackerr.conf
```

**Comment by davidnewhall** [comment]
Folks really struggle getting the ownership of their files and folders correct. Would be nice if there was a better proxmox guide for making unpackerr work with Sonarr and Radarr. [This one](https://forum.proxmox.com/threads/how-to-set-up-unpackerr-utility-in-lxc-with-torrent.130054/) leaves a lot to be desired.
